### DISCUSSION TITLE
Ashwin Discussion

### DISCUSSION THREAD

**Discussion by JeremyFriesenGitHub** [cuhacking/2025#98] [Ideas]
If you have any questions, you can post them here.

**Comment by AshwinSri23** [comment]
In an earlier comment, you had mentioned that there would be a meeting sometime tomorrow. When and where would the meeting be taking place?

**Reply by JeremyFriesenGitHub** [reply]
I'm pretty sure it's from 3:45 pm to 5:45 pm at the library I believe (I don't know which floor). @JowiAoun or @HasithDeAlwis could probably provide the details on that. If that time doesn't work, we could always schedule a call at a later time and/or date if you prefer.

**Reply by AshwinSri23** [reply]
I won't be able to make it to the meeting later today. However, I don't mind scheduling a call at around that same time on Friday(tomorrow). Additionally, I would be available on Saturday and any time before 5PM on Sunday.

**Reply by HasithDeAlwis** [reply]
Hey @AshwinSri23 I can schedule a call with you and I can go over what happened in the call if you'd like. Also we're going to making issues and updating the project board during the meeting, so you can look over those too if you'd like as well!

**Reply by AshwinSri23** [reply]
Sounds good. 
Thank you @HasithDeAlwis.
How would I be contacted about a set call date and time?
